docs/specs/config.md

```diff
@@ -33,28 +36,21 @@
 [op.json](https://github.com/MicrosoftDocs/azure-docs-pr/blob/master/.openpublishing.publish.config.json) [docfx.json](https://github.com/MicrosoftDocs/azure-docs-pr/blob/master/docfx.json)
 
 ``` yml
-name: azure-documents
-product: Azure
-defaults: msdocs.yml
+name: Azure.azure-documents
+default: msdocs.yml
```

Comment:
*eslint* is using [**extends**](https://eslint.org/docs/user-guide/configuring) to represent shareable configs. And it supports array form

Reply:
Sounds good. Rename to `extend` for consistency with other name like `include`, `exclude`.

---
In reply to: [182989232](https://github.com/dotnet/docfx/pull/2644#discussion_r182989232) [](ancestors = 182989232)